xtask/src/commands/test.rs

```diff
@@ -55,11 +78,12 @@ impl Test {
                 continue;
             }
 
-            eprintln!("Running tests with features: {}", features.join(", "));
```

Comment:
I liked the long sentence (with capital letter at the beginning) better

Reply:
oh indeed, me too!